Formula/dvorak7min.rb

```diff
@@ -1,8 +1,7 @@
 class Dvorak7min < Formula
   desc "Dvorak (simplified keyboard layout) typing tutor"
   homepage "https://web.archive.org/web/dvorak7min.sourcearchive.com/"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/d/dvorak7min/dvorak7min_1.6.1+repack.orig.tar.gz"
-  mirror "https://mirrors.kernel.org/debian/pool/main/d/dvorak7min/dvorak7min_1.6.1+repack.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/d/dvorak7min/dvorak7min_1.6.1+repack.orig.tar.gz"
```

Comment:
There were two `brew audit` failures here, both for `deb.debian.org`. I decided to change the `url` stanza to `deb.debian.org` and remove the `mirror` stanza otherwise they'd be duplicates.

Reply:
Yes, that's good.